src/components/VideoDetail/VideoInfo/VideoInfo.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { parseDatetime } from '../../../utils/parseDatetime';
+import {Title, ChannelAndDatePublished, Separator, DescriptionContainer, DescriptionText, ShowLessMore } from './VideoInfo.styles';
+// const mockVideoData = require('../../../mock-data/youtube-video-api.json').items[0];
+
+const variants = {
+    open: {height: "auto"},
+    closed: {height: 50}
+}
+
+export const VideoInfo = ({video}) => {
+    const [isOpen, setIsOpen] = useState(false);
+    if(video){
+        const {title, channelTitle, publishedAt, description} = video.snippet;
+
+        return <div>
+            <Title>{title}</Title>
+            <ChannelAndDatePublished>{`${channelTitle} • ${parseDatetime(publishedAt)}`}</ChannelAndDatePublished>
+            <Separator></Separator>
+            <DescriptionContainer animate={isOpen ? "open":"closed"} variants={variants}>
+                <DescriptionText>{description}</DescriptionText>
+            </DescriptionContainer>
+            <ShowLessMore onClick={()=>setIsOpen(!isOpen)}>Show {!isOpen && 'more'} {isOpen && 'less'}</ShowLessMore>
+        </div>
+    }
+    return <></>
```

Comment:
Here you have a prop called `video`, and you have an `if` statement to check if the prop in truthy, in this case the parent is the one that should check that prop, for example:

```jsx
function Parent() {
  const video = getFromSomewhere();

  return (
    <div>
      {video && <VideoInfo video={video} />}
    </div>
  );
}
```

There is no reason to return an empty fragment, if you use the suggestion you have to update the places where you use the component and check if the prop is truthy before trying to render the `VideoInfo`

```suggestion
export const VideoInfo = ({video}) => {
    const [isOpen, setIsOpen] = useState(false);
    const {title, channelTitle, publishedAt, description} = video.snippet;

    return (
      <div>
          <Title>{title}</Title>
          <ChannelAndDatePublished>{`${channelTitle} • ${parseDatetime(publishedAt)}`}</ChannelAndDatePublished>
          <Separator></Separator>
          <DescriptionContainer animate={isOpen ? "open":"closed"} variants={variants}>
              <DescriptionText>{description}</DescriptionText>
          </DescriptionContainer>
          <ShowLessMore onClick={()=>setIsOpen(!isOpen)}>Show {!isOpen && 'more'} {isOpen && 'less'}</ShowLessMore>
      </div>
    );
```